src/dapps/aave-dapp/components/AaveSummary.vue

```diff
@@ -156,6 +157,15 @@ export default {
     }
   },
   computed: {
+    actionTitle() {
+      if (this.isBorrow) {
+        return 'Borrow';
+      }
+      if (this.isRepay) {
+        return 'Repay';
+      }
+      return 'Deposit';
+    },
```

Comment:
Do we need to include isCollateral in this / is isCollateral used anywhere?  Or will Deposit always be the default state.

Reply:
we don't because its hidden if its collateral! but good call